### PR TITLE
Minor doc changes

### PR DESCRIPTION
**Pull request by metc**
Hello mbed/BLE team,

This is my first (minor) contribution. I fixed some broken links in the docs.

### CODE REVIEW

**Comment by ciarmcom** [comment]
Automatic CI verification build not done, please verify manually.

**Comment by metc** [comment]
I signed the CLA with a different email address I used to push these commits. It is fixed now, maybe this was a problem... Please let me know.
@ciarmcom rebase

File: README.md [hunk 1]
```diff
@@ -1,21 +1,25 @@
 # mbed Bluetooth Low Energy Stack
-This is the github repo for the BLE_API used by developer.mbed.org. Please see [mbed BLE Homepage](http://developer.mbed.org/teams/Bluetooth-Low-Energy/) for all documentation, code examples and general help.
+This is the Github repo for the `BLE_API` used by developer.mbed.org. Please see the [mbed BLE Homepage](https://developer.mbed.org/teams/Bluetooth-Low-Energy/) for all documentation, code examples and general help.
 
 # Supported Services
-Supported GATT services and constantly being added and can be found in the /services folder.
-Currently supported  services include:
-* Battery 
+Supported GATT services and constantly being added and can be found in the `ble/source/services/` folder.
```
Comment (rgrover) [inline README.md]: services are under `ble/ble/services`. 
Thanks so much for fixing these links.

**Comment by rgrover** [comment]
thanks. I'll accept this and fix the minor thing about the path to services.

**Pull request merged**